package.json

```diff
@@ -46,7 +46,7 @@
   "keywords": [
     "Scuttling"
   ],
-  "author": "JArnstein (c) 2015",
+  "author": "JArnstein © 200BC to (new Date().getFullYear() + 1)",
```

Comment:
This is a string, therefore `new Date().getFullYear() + 1` does not evaluate to anything.